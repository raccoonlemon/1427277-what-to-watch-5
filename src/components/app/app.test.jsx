import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import App from "./app";

const renderer = new ShallowRenderer();

it(`<App> renders correctly`, () => {
  renderer.render(<App/>);
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});
