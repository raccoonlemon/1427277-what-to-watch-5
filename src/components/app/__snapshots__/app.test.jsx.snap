// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> renders correctly 1`] = `
<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Switch>
    <Route
      exact={true}
      path="/"
    >
      <Memo(Connect(MainScreen)) />
    </Route>
    <Route
      exact={true}
      path="/login"
    >
      <Memo(Connect(SignInScreen)) />
    </Route>
    <Memo(Connect(PrivateRoute))
      exact={true}
      path="/mylist"
      render={[Function]}
    />
    <Memo(Connect(PrivateRoute))
      exact={true}
      path="/films/:id/review"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/films/:id"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/player/:id"
      render={[Function]}
    />
    <Route
      render={[Function]}
    />
  </Switch>
</Router>
`;
